client/pages/gameover/gameover.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, memo } from "react";
+import block from "bem-cn";
+import { Page } from "../../components/page";
+import { Text } from "../../components/text";
+import { Heading } from "../../components/heading";
+import "./error.css";
+import { LinkView } from "../../components/link-view";
+
+const b = block("error-page");
+const maxLenght = 6;
+const score = 100;
+const highscore = 200;
+
+type Props = {
+  className?: string;
+};
+
+const GameOver: FC<Props> = ({ className = "" }) => {
```

Comment:
Когда функция просто возвращает значение, можно обойтись без return:
```suggestion
const GameOver: FC<Props> = ({ className = "" }) => (<Page/>}
```